rnd/autogpt_builder/src/components/edit/control/SaveControl.tsx

```diff
@@ -51,8 +51,10 @@ export const SaveControl = ({
 
   return (
     <Popover>
-      <PopoverTrigger className="hover:bg-gray-100 hover:text-gray-900 dark:hover:bg-gray-800 dark:hover:text-gray-50 dark:text-white">
-        <Save className="size-4" />
+      <PopoverTrigger asChild>
```

Comment:
Our Popover Trigger here acts as a button here. Since the pop-over trigger itself has a set of styles by default you might see some style issues by removing them here.

Reply:
The `PopoverTrigger` does not have inherent `Button` styling, and the example in the [documentation](https://ui.shadcn.com/docs/components/popover) uses `<PopoverTrigger asChild> <Button ...>`.